actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1586,6 +1662,11 @@ def fields_for(record_name, record_object = nil, fields_options = {}, &block)
         @template.fields_for(record_name, record_object, fields_options, &block)
       end
 
+      # TODO: Documentation
+      def fields(scope = nil, model: nil, **options, &block)
+        fields_for(scope || model, model, **options, &block)
+      end
```

Comment:
@dhh added the fields method to do:

```ruby
form_with(model: @post) do |form|
  form.fields(:comment, model: @comment) do |fields|
    fields.text_field # ...
  end
end
```

But it reads a bit weird if the passed `model` is a collection: `fields(:comments, model: @post.comments)`.

Not sure if you intended this to be much more than a `fields_for` alias with the don't output ids and classes by default config.